src/invoice2data/main.py

```diff
@@ -117,6 +117,13 @@ def create_parser():
         help='Choose output format. Default: none',
     )
 
+    parser.add_argument(
+        '--output-date-format',
+        dest='output_date_format',
+        default='%d/%m/%Y',
+        help='Choose output date format. Default: %d/%m/%Y',
```

Comment:
Would suggest to use ISO 8601 (2019-02-22), even if it breaks stuff for some people.

Reply:
OK!